README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### ABOUT ME
+My name is Gabrielle Bogert, I am fourth year majoring in Public Policy and minoring in Environmental and Urban Studies. I was born and raised in Miami, the middle child of three girls. 
+**I'm passionate about**
+* American politics
+* Climate change mitigation and adaption
+* Digital media and communication
+* Public transportation (yes I'm a NUMTOT)
+* Libraries
+* Listening to, sharing, and discovering music
+* Scrolling through Twitter and TikTok for hours
+* Going down Wikipedia rabbit holes from the hours of 1 to 4 AM each night of quarantine
+* Reinventing myself constantly by adding hobbies and skills to the institutional memory of my physical form...we're currently on baking
+
+*The last thing that made me laugh is*
+![Funny Tweet](Desktop/hahaha.png)
```

Comment:
If you don't commit the image (hahaha.png) relative paths like this won't render on Github.